README.md

```diff
@@ -114,6 +124,16 @@ This event is triggered after a booking has been cancelled on the HealthEngine p
 }
 ```
 
+If the booking is associated with a corporate group (directly via the booking model, or via the booking practice), we conditionally attach a corporate group fragment:
```

Comment:
I think these docs are supposed to be for third parties so I don't think this wording would make sense to them

Reply:
oh I was not aware of that, agreed, hmmmm

Reply:
perhaps - ~ there may be a `corporate_group` object attached if the related practice has been setup to specify a corporate structure - ?

thoughts? prayers?